### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Daf98**
This is my first Capstone project. Some of the things I implemented:
- Made a home page and about page for both mobile and desktop screen sizes
- Used CSS for styling
- Created a logo for my page
- Added a toggable menu with Javascript
- Created the "Speaker" cards using dynamic HTML

I hope you like it 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+- Twitter: [@dafne_azzolina](https://twitter.com/dafne_azzolina)
+- LinkedIn: [dafne-azzolina](https://www.linkedin.com/in/dafne-azzolina/)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (gemmen29) [inline README.md:46]: - [x] Kindly add an Acknowledgements to the author [Cindy Shin](https://www.behance.net/adagio07) as it's required by the [Creative Commons License](https://creativecommons.org/licenses/).as required by the [Creative Commons License](https://creativecommons.org/licenses/).
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Daf98, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Great UI Design ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Daf98) [reply README.md:46]: I forgot! Thank you!

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Daf98 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous review.

**Optional**
- Please fix the MIT link for a completely professional README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**